package.json

```diff
@@ -6,7 +6,7 @@
     "start": "yarn build -w",
     "start:example:js": "yarn build && npx http-server ./packages/js -o /example",
     "lint": "eslint . --ext .ts,.tsx",
-    "test": "jest",
+    "test": "yarn build && jest",
```

Comment:
I think this should be moved to a `pre` hook (so `pretest` in this case), just for better debugging in case one of the scripts encounters an error

Reply:
Would it even make sense to run the build before the tests though? Doesn't Jest use the source files directly for testing purposes?

Reply:
@chrisvanmook Good point

@jonkoops I was thinking so too, but looks like not. I haven't tried with #197

Reply:
@jonkoops because `matomo-tracker-react` uses `matomo-tracker-js` it will start complaining

Reply:
I've moved `yarn build` to `pretest` in f905a2462eefc73eaead59a9972a81fddaca2c9e